src/ExtendedGridOrbitals.cc

```diff
@@ -551,12 +551,12 @@ void ExtendedGridOrbitals::multiply_by_matrix(
     prod_matrix_tm_.stop();
 }
 
-#ifdef HAVE_MAGMA
 template <>
 void ExtendedGridOrbitals::multiply_by_matrix(const ReplicatedMatrix& matrix)
 {
     prod_matrix_tm_.start();
 
+#ifdef USE_MAGMA
```

Comment:
Should this be HAVE_MAGMA instead? Also used in replicatedMatrix.cc

Reply:
Fixed